ParkingDatabase/src/main/java/com/example/parkingdatabase/api/RestCarController.java

```diff
@@ -0,0 +1,47 @@
+package com.example.parkingdatabase.api;
+
+import com.example.parkingdatabase.dao.entity.Car;
+import com.example.parkingdatabase.manager.ParkingServiceManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/car")
+public class RestCarController {
```

Comment:
This controller looks way better than CarController to me, let's invest into this class